nixos/modules/services/security/keylime-agent.nix

```diff
@@ -0,0 +1,83 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let
+  cfg = config.services.keylime-agent;
+in
+{
+  options = {
+    services.keylime-agent = {
+      enable = mkOption {
+        default = false;
+        type = types.bool;
+        description = lib.mdDoc "Whether to enable keylime-agent service.";
+      };
+
+      stateDir = mkOption {
+        type = types.path;
+        default = "/var/lib/keylime/";
+        defaultText = literalExpression "/var/lib/keylime/";
+        description = lib.mdDoc "State directory for keylime";
+      };
```

Comment:
Is there any reason why people would like to keep this elsewhere than `/var/lib/keylime`? Elsewise, we could maybe just use systemd's `StateDirectory`